Packs/CommonTypes/IncidentFields/incidentfield-Caller.json

```diff
@@ -0,0 +1,42 @@
+{
+	"associatedToAll": true,
```

Comment:
@Itay4 It's either associating them to all and placing them under CommonTypes or opening a new pack for ticketing systems fields, @anara123 what is standard here? (it's possible of course that in the future other non-ticketing integrations will have a caller field for example so if it's in a ticketing pack it will need to move)